esphomeRemote/esphomeRemote.h

```diff
@@ -915,6 +994,8 @@ bool selectRootMenu() {
   case bootMenu:
     ESP_LOGD("WARNING", "menu is bad  %d", menuIndex);
     return false;
+  default:
+    break;

```

Comment:
i prefer not to add defaults to the switch statements so it's easier to spot when adding new cases

Reply:
Oh yeah I understand 👍